Library/Homebrew/sandbox.rb

```diff
@@ -231,17 +231,14 @@ def exec(*args)
     end
   end
 
-  private
-
-  sig { params(path: Pathname).returns(Pathname) }
-  def expand_realpath(path)
-    raise unless path.absolute?
-
-    path.exist? ? path.realpath : expand_realpath(path.parent)/path.basename
-  end
-
+  # @api private
   sig { params(path: T.any(String, Pathname), type: Symbol).returns(String) }
   def path_filter(path, type)
+    invalid_char = ['"', "'", "(", ")", "\n"].find do |c|
+      path.to_s.include?(c)
+    end
```

Comment:
Nitpick, nonblocking: could be `String#match?` with a regexp instead.

Reply:
punting for now